dev/com.ibm.ws.jpa_22_fat/publish/servers/JPA22InjectionServer/server.xml

```diff
@@ -0,0 +1,38 @@
+<!--
+    Copyright (c) 2017 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="JPA 22 Injection Server">
+
+	<featureManager>
+		<feature>servlet-4.0</feature>
+		<feature>jpa-2.2</feature>
+	    <feature>componenttest-1.0</feature>
+    </featureManager>
+    
+    <include location="../fatTestPorts.xml"/>
+    
+    <dataSource id="DefaultDataSource" fat.modify="true">
```

Comment:
setting fat.modify="true" isn't necessary unless you're calling `server.configureForAnyDatabase()`, which isn't yet supported in OpenLiberty

Reply:
eventually we're planning to support it yes, although we might go with a different approach besides fat.modify